extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/ESH-INF/config/config.xml

```diff
@@ -14,6 +14,41 @@
 			<description>The MAC Address used for communication to this device.
 			</description>
 		</parameter>
+		<parameter name="appKey" type="text" required="false">
+			<label>Authorization Key</label>
+			<description>An authorization key used to identify the client application.
+						Should be requested from the developer portal.
+			</description>
+		</parameter>
+		<parameter name="notificationVolume" type="integer" min="10" max="70" step="1" unit="%" required="false">
```

Comment:
Hm, looking at this, might I come up with an idea?
All those 4 parameters are specific to the notificationSound channel - they are not useful for anything else.
So imho, it would make a lot of sense to move them to the channel as a configuration and not have them on the thing. The cool thing is, that we could then mark the Thing extensible by "notificationSound" channels, so that the user can add multiple channels of that type to his Thing - each having a different configuration, so you could have different volumes, texts, etc. for different scenarios. Wdyt?